examples/async/babel.config.js

```diff
@@ -5,15 +5,16 @@ const cjs = BABEL_ENV === 'cjs' || NODE_ENV === 'test'
 module.exports = {
   presets: [
     [
-      '@babel/env',
+      '@babel/preset-env',
       {
         loose: true,
         modules: false,
         forceAllTransforms: true,
       },
     ],
-    '@babel/react',
-    ['@babel/stage-2', { decoratorsLegacy: true }],
+    '@babel/preset-react',
   ],
-  plugins: [cjs && '@babel/transform-modules-commonjs'].filter(Boolean),
+  plugins: [cjs && '@babel/plugin-transform-modules-commonjs', '@babel/plugin-proposal-object-rest-spread'].filter(
```

Comment:
isnt object-rest-spread included in preset-env? (not sure if its on by default or not though)

also - it accepts `loose` flag too and we should use it

great change about removing `stage-2` 👌

Reply:
`object-rest-spread` is still in [stage-3](https://babeljs.io/docs/en/babel-preset-stage-3), so we need to keep it in the plugin list.

I'll add `loose` flag. And just out of curiosity, what's the benefits of this flag?